application/lib/model/Task.dart

```diff
@@ -6,10 +10,44 @@ enum Status {
 
 enum Priority { low, medium, high, critical }
 
-class Task {
-  Task({this.taskName, this.taskStatus, this.taskPriority, this.newInformation = false});
+class Task implements Json {
+  Task({this.taskName, this.appId, this.screenId, this.taskDescription = ''});
+
+  @override
+  Task.fromJson(Map<String, dynamic> json) {
+    taskName = json['taskName'];
+    appId = json['appId'];
+    taskDescription = json['description'];
+    String test2 = jsonEncode(json['screenId']).replaceAll('[', '').replaceAll(']', '');
```

Comment:
Der er ikke brug for test2, og linteren bliver også glad hvis du fjerner den :D 
```suggestion
    jsonEncode(json['screenId']).replaceAll('[', '').replaceAll(']', '');
```